test/signalrclient-e2e-tests/hub_connection_tests.cpp

```diff
@@ -181,4 +181,127 @@ TEST(hub_connection_tests, send_message_after_reconnect)
     ASSERT_FALSE(received_event->wait(200));
 
     ASSERT_EQ(*message, U("[\"Send: test\"]"));
-}
\ No newline at end of file
+}
+
+TEST(hub_connection_tests, send_message_empty_param)
+{
+    auto hub_conn = std::make_shared<signalr::hub_connection>(url + U("SignalR"));
```

Comment:
for hub connections `SignalR` should be added automatically. Aren't you seeing this?